### PR TITLE
disable build isolation which causes problems for windows

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Why am I running into such strange error only on Python 3.7 for windows.
https://github.com/numpy/numpy/issues/12016

**Comment by hmaarrfk** [comment]
@conda-forge/python @conda-forge/core  @conda-forge/scikit-image 

In previous builds, it seems that windows was pulling in numpy from pypi as outlined here
https://github.com/conda-forge/scikit-image-feedstock/issues/36#issuecomment-450786253

Although windows failed on 3.7. I'm pretty inclined to merge this in.

1. People on 3.7 are probably using modern numpy, so they are likely not running into this issue
2. It will fix the issue for all those running 3.6 and older versions of numpy.

If somebody wants to tag in for windows, that would be greatly appreciated!!!!

**Comment by hmaarrfk** [comment]
@conda-forge/scikit-image does anybody have any ideas why the builds are failing on windows.

I've tried:
1. Downloading the sources from github (they don't have any cythonized files).
2. Downgrading the version of cython back to 0.28

It builds on my own computer, but not Appveyor

**Comment by jni** [comment]
@jakirkham is probably the best person to answer this... the subprocess call to bld.bat is failing, but no clues as to why. @jakirkham is there a way to ensure that the stdout and stderr from that call get printed on failure? @hmaarrfk if not, that might be a good contribution to make to conda build.

```pytb
Command "C:\bld\scikit-image_1546480104042\_h_env\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\appveyor\\AppData\\Local\\Temp\\1\\pip-req-build-_pr74h7r\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\appveyor\AppData\Local\Temp\1\pip-record-lecq_zi1\install-record.txt --single-version-externally-managed --compile" failed with error code 3221225477 in C:\Users\appveyor\AppData\Local\Temp\1\pip-req-build-_pr74h7r\
Exception information:
Traceback (most recent call last):
  File "C:\bld\scikit-image_1546480104042\_h_env\lib\site-packages\pip\_internal\cli\base_command.py", line 143, in main
    status = self.run(options, args)
  File "C:\bld\scikit-image_1546480104042\_h_env\lib\site-packages\pip\_internal\commands\install.py", line 366, in run
    use_user_site=options.use_user_site,
  File "C:\bld\scikit-image_1546480104042\_h_env\lib\site-packages\pip\_internal\req\__init__.py", line 49, in install_given_reqs
    **kwargs
  File "C:\bld\scikit-image_1546480104042\_h_env\lib\site-packages\pip\_internal\req\req_install.py", line 791, in install
    spinner=spinner,
  File "C:\bld\scikit-image_1546480104042\_h_env\lib\site-packages\pip\_internal\utils\misc.py", line 705, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pip._internal.exceptions.InstallationError: Command "C:\bld\scikit-image_1546480104042\_h_env\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\appveyor\\AppData\\Local\\Temp\\1\\pip-req-build-_pr74h7r\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\appveyor\AppData\Local\Temp\1\pip-record-lecq_zi1\install-record.txt --single-version-externally-managed --compile" failed with error code 3221225477 in C:\Users\appveyor\AppData\Local\Temp\1\pip-req-build-_pr74h7r\
Traceback (most recent call last):
  File "C:\Miniconda36-x64\Scripts\conda-build-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\cli\main_build.py", line 438, in main
    execute(sys.argv[1:])
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\cli\main_build.py", line 429, in execute
    verify=args.verify)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\api.py", line 201, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\build.py", line 2204, in build_tree
    notest=notest,
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\build.py", line 1394, in build
    windows.build(m, build_file, stats=build_stats)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\windows.py", line 316, in build
    check_call_env(cmd, cwd=src_dir, stats=stats)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\utils.py", line 313, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\utils.py", line 293, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['cmd.exe', '/c', 'bld.bat']' returned non-zero exit status 1.
```

macOS is also failing on Travis, also not sure why:

https://travis-ci.org/conda-forge/scikit-image-feedstock/jobs/474665631#L253-L256

```pytb
Traceback (most recent call last):
  File "/Users/travis/miniconda3/bin/setup_conda_rc", line 7, in <module>
    from conda_forge_ci_setup.build_utils import setup_conda_rc
ModuleNotFoundError: No module named 'conda_forge_ci_setup'
```

**Comment by hmaarrfk** [comment]
@jni, thanks for chiming in. How do you feel about merging this now, as is, it would fix things on windows for python 2.7 and 3.6????

mac osx errors seem to be related to caching on the servers. It has been happening CI wide. Not too worried about those.

**Comment by jni** [comment]
@hmaarrfk what do you mean "CI wide"? All conda-forge?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -5,8 +5,11 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/s/scikit-image/scikit-image-{{ version }}.tar.gz
-  sha256: 86a9b3b4f74f231e0a6bcfd3235dcf3f0118df25dac21201da5e064d681e2c50
+  # url: https://pypi.io/packages/source/s/scikit-image/scikit-image-{{ version }}.tar.gz
+  # sha256: 86a9b3b4f74f231e0a6bcfd3235dcf3f0118df25dac21201da5e064d681e2c50
+  url: https://github.com/scikit-image/scikit-image/archive/v{{ version }}.tar.gz
+  fn: scikit-image-{{ version }}.tar.gz
+  sha256: 8da6fb09aeefb757735c510650ac0072be3831fa76d9747285f3c6ea1e0c5a08
```
Comment (jni) [inline recipe/meta.yaml]: Why are the sha hashes different on GH and PyPI???

**Comment by hmaarrfk** [comment]
> @hmaarrfk what do you mean "CI wide"? All conda-forge?

It seems to be isolated to OSX. OSX seems like it is installing 5 month old packages.

**Comment by jni** [comment]
@hmaarrfk I would prefer if we keep PyPI as the source. Would that be fine, given that this PR doesn't fix the situation on 3.7? Or do the other Python versions also fail without that change?

File: recipe/meta.yaml (hunk 1 above)
Comment (hmaarrfk) [reply recipe/meta.yaml]: pypi includes cythonized files. and probably generated with "python setup.py sdist"

**Comment by jni** [comment]
@hmaarrfk 

> It seems to be isolated to OSX.

Yes but OSX for everyone or OSX for NumPy dependent packages or OSX for us?

**Comment by hmaarrfk** [comment]
Anyway, @jakirkham thinks that this kind of stuff should have been resolved internally in conda-forge, but something might have slipped
https://gitter.im/conda-forge/conda-forge.github.io?at=5c2d782937975e7ca965954b

**Comment by jni** [comment]
Coolio, if CI passes, minus the expected failures, do feel free to merge. 👍 Thanks yet again for all this painful but essential build work!

**Comment by hmaarrfk** [comment]
So I only care if windows 2.7 and 3.6 pass.
Windows 3.7 and OSX won't get rebuilt, but it really doesn't matter. (IMO)

**Comment by hmaarrfk** [comment]
The build got futher this time?????? I have a feeling it is related to the specific version of the compiler that appveyor may or may not have upgraded to :/

Could be related to this.
https://github.com/rust-lang/rust/issues/50604

**Comment by jakirkham** [comment]
Here's the [relevant bit about Travis CI's macOS builds]( https://gitter.im/conda-forge/conda-forge.github.io?at=5c2d3da12863d8612bcbbe6b ). It looks like an edge caching issue, which it sounds like Anaconda had some issues with today. Probably gets resolved tomorrow.

> Anyway, @jakirkham thinks that this kind of stuff should have been resolved internally in conda-forge, but something might have slipped

To be clear, that is something `conda-build` does.

**Comment by hmaarrfk** [comment]
Ok, i'm going to merge this, and start a new CI run just for windows.

**Pull request merged**